src/display/annotation_layer.js

```diff
@@ -728,6 +777,7 @@ class TextWidgetAnnotationElement extends WidgetAnnotationElement {
         element.type = "text";
         element.setAttribute("value", textContent);
       }
+      GetElementsByNameSet.add(element);
```

Comment:
Could the `name` assignment be moved from below to this location, so that it is more obvious to which name this element is associated? That would also be more consistent with the other 3 initializations of `GetElementsByNameSet`.
https://github.com/mozilla/pdf.js/blob/6040ae32819c454d8e1724c9d298e94b64e76f67/src/display/annotation_layer.js#L954